### ISSUE TITLE
1.3 beta Cordova iOS fastclick problem with contenteditable=“true”

### ISSUE THREAD

**Issue by jhatchr** [meteor/meteor#6286]
iOS fails to allow the user to tap into contenteditable elements which seems to be an issue with fastclick. Input and other fields work as expected. On the mobile web version contenteditable=“true” also behaves as expected.

The experience was improved a bit by adding the fastclick class below to `onRendered` in a global location, however, it’s still problematic - particularly for the case of a contenteditable Rich Text Editor.

```
$('[contenteditable="true"]').each(function() {
  $(this).addClass('needsclick');       
}); 
```

**Comment by hwillson** [comment]
Yes, this is a `fastclick` bug. A fix was submitted to the main `fastclick` repo (https://github.com/ftlabs/fastclick/pull/381) more than 2 years ago, but was never accepted. Actually, `fastclick` itself is pretty much no longer supported / maintained. The last released version was 1.0.6 (January 2015), and there are [years of PR's pending](https://github.com/ftlabs/fastclick/pulls). The main reason for this is that most modern mobile browsers have removed the 300ms touch delay that `fastclick` works around.

That being said, there are still some browsers with the 300ms delay, and since `fastclick` is installed alongside every new Meteor app (via the [`mobile-experience`](https://github.com/meteor/meteor/tree/devel/packages/mobile-experience) package), we should probably come up with a plan to either update it and support a custom version, or deprecate it and decide on an alternative approach. Maybe the time has come to deprecate its use with Meteor (since both major mobile browsers, Safari and Chrome, no longer need it), and simply document (in the Guide) how it can be used if needed (installed directly from npm). Food for thought anyways - more discussion is definitely needed ...

**Comment by hwillson** [comment]
This issue was discussed during this weeks contributor call. Since `fastclick` is no longer needed with  modern browsers, the current plan is to:

1. Remove the `fastclick` dependency from `mobile-experience`.
2. Deprecate the `fastclick` package (moving it into `deprecated`).
3. Deploy these changes as a `mobile-experience` package patch.

After this happens, anyone who is still interested in using `fastclick` can do so by including it directly (e.g. from npm).
